data/json/items/comestibles.json

```diff
@@ -3355,10 +3364,11 @@
         "name" : "vegetable aspic",
         "weight" : 120,
         "color" : "green",
+        "spoils_in" : 720,
         "comestible_type" : "FOOD",
         "symbol" : "%",
         "nutrition" : 20,
-        "description" : "A dish in which vegetables are set into a gelatin made from a plant stock.  This food will never spoil.",
+        "description" : "A dish in which vegetables are set into a gelatin made from a plant stock.  This food will last long.",
```

Comment:
"This food will last a long time." sounds better.

Reply:
Agreed. Will implement shortly.